compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +64,25 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getPeelAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, "enable_loop_peeling");
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface *funcOp) {
+  auto peelAttrName = getPeelAttrName(funcOp->getContext());
+  auto trueAttr = BoolAttr::get(funcOp->getContext(), true);
+
+  IREE::Codegen::TranslationInfoAttr translationInfo =
+      getTranslationInfo(*funcOp);
+  DictionaryAttr config = translationInfo.getConfiguration();
+  if (config) {
+    auto peelAttr = translationInfo.getConfiguration().getNamed(peelAttrName);
+    if (peelAttr && peelAttr->getValue() == trueAttr) {
+      return true;
+    }
+  }
```

Comment:
nit: early exists (less nesting)
```c++
if (!config)
   return false;
```

Reply:
(similar naming nits here this is `enableLoopPeelingAttrName`)

Reply:
That's much cleaner, thanks!